tasks/diff.py

```diff
@@ -4,256 +4,18 @@
-                    platform, arch = combo.split("/")
-                    goos, goarch = GOOS_MAPPING.get(platform), GOARCH_MAPPING.get(arch)
-                    target = f"{binary}-{goos}-{goarch}"
-                    prettytarget = f"{binary} {goos}/{goarch}"
-
-                    if target in diffs:
-                        targetdiffs = diffs[target]
-                        add, remove = patch_summary(targetdiffs)
-
-                        if report_metrics:
-                            tags = [
-                                f"build:{build}",
-                                f"flavor:{flavor.name}",
-                                f"os:{goos}",
-                                f"arch:{goarch}",
-                                f"git_sha:{commit_sha}",
-                                f"git_ref:{git_ref}",
-                            ]
-
-                            if git_ref:
-                                tags.append(f"git_ref:{git_ref}")
-
-                            dependency_diff = create_count(METRIC_GO_DEPS_DIFF, timestamp, (add - remove), tags)
-                            send_metrics([dependency_diff])
-
-                        color_add = color_message(f"+{add}", "green")
-                        color_remove = color_message(f"-{remove}", "red")
-                        print(f"== {prettytarget} {color_add}, {color_remove} ==")
-                        print(f"{color_patch(targetdiffs)}\n")
-
-                        summary = f"<summary>+{add}, -{remove}</summary>"
-                        diff_block = f"<pre lang='diff'>\n{targetdiffs}\n</pre>"
-                        pr_comment.append(
-                            f"<tr><td>{binary}</td><td>{goos}</td><td>{goarch}</td><td><details>{summary}\n{diff_block}</details></td></tr>"
-                        )
-                    else:
-                        print(f"== {prettytarget} ==\nno changes\n")
-
-            pr_comment.append("</tbody></table>")
-            if report_file:
-                with open(report_file, 'w') as f:
-                    f.write("\n".join(pr_comment))
-        else:
-            print("no changes for all binaries")
-            if report_file:
-                # touch file
-                open(report_file, 'w').close()
-
-
-def color_patch(diff):
-    out = []
-    for line in diff.splitlines():
-        if line.startswith("+"):
-            out.append(color_message(line, 'green'))
-        elif line.startswith("-"):
-            out.append(color_message(line, 'red'))
-        else:
-            out.append(line)
-    return "\n".join(out)
-
-
-def patch_summary(diff):
-    add_count, remove_count = 0, 0
-    for line in diff.splitlines():
-        if line.startswith("+"):
-            add_count += 1
-        elif line.startswith("-"):
-            remove_count += 1
-    return add_count, remove_count
+def go_deps(ctx):
+    raise Exit("This task has been renamed `go-deps.diff`")
```

Comment:
Since the task `diff.go-deps` was primarily used in CI, we could probably remove this fallback code 😄 
That way, we do not have to worry about removing this empty task in the future

Reply:
Yeah I considered it too, but I actually run that task locally so I'd rather make it explicit in case someone else also does 😄